### PR TITLE
security: support init tls config from cert/key/ca bytes

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve? 

to slove https://github.com/pingcap/dm/issues/1909, we need  support init tls config from cert/key/ca bytes

### What is changed and how it works?

support init tls config from cert/key/ca bytes

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch
 - Need to be included in the release note

### CODE REVIEW

File: pkg/utils/security_test.go [hunk 1]
```diff
@@ -160,3 +161,21 @@ func runServer(ctx context.Context, tlsCfg *tls.Config, port int, c *C) *http.Se
 func getTestCertFile(dir, role string) (string, string, string) {
 	return path.Join(dir, "ca.pem"), path.Join(dir, fmt.Sprintf("%s.pem", role)), path.Join(dir, fmt.Sprintf("%s.key", role))
 }
+
+func loadTLSContent(caPath, certPath, keyPath string) (caData, certData, keyData []byte) {
```
Comment (lance6716) [inline pkg/utils/security_test.go]: pass a `C` into this function, and use its checking method

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by lance6716 — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**